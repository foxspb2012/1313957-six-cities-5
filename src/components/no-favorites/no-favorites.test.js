import React from 'react';
import NoFavorites from './no-favorites';
import renderer from 'react-test-renderer';

it(`Should NoFavorites render correctly`, () => {
  const tree = renderer
    .create(
        <NoFavorites />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
