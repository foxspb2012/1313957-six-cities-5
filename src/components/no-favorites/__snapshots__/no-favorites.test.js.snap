// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NoFavorites render correctly 1`] = `
<div
  className="favorites__status-wrapper"
>
  <b
    className="favorites__status"
  >
    Nothing yet saved.
  </b>
  <p
    className="favorites__status-description"
  >
    Save properties to narrow down search or plan yor future trips.
  </p>
</div>
`;
